### ISSUE TITLE
Server launch on demand for Unity apps

### ISSUE THREAD

**Issue by Bundenth** [OSVR/OSVR-Unity#173]
Hello,

Are there any plans of making the server launch on demand for Unity apps to avoid the need of manually running the server in parallel? I understand Unity supports many platforms -android, Windows, Mac...- and this may be something that needs to be delegated to the OSVR plugin on its respective platform, but reducing the barriers for end users would make OSVR much more accessible and usable.

**Comment by DuFF14** [comment]
This is already done in a PR here: https://github.com/OSVR/OSVR-Unity/pull/157

I was asked not to merge it in yet as it affects some ongoing work with Android. Not sure what the current status is there. Perhaps this could be merged if it was limited to PC only. Regardless, you could test this out to see if it works and comment in the PR. If more people want it merged, the faster it will happen.

 There was also a bug where you have to remove a backslash from the OSVR environment variable that is set by the installer. Not sure if thats still the case.

**Comment by DuFF14** [comment]
Merged, will work with the next OSVR-Core build, on Windows only.
